RTProtocol.cpp

```diff
@@ -1471,210 +1470,210 @@ int CRTProtocol::ReceiveRTPacket(CRTPacket::EPacketType &eType, bool bSkipEvents
 }
 
 
-CNetwork::ResponseType CRTProtocol::Receive(CRTPacket::EPacketType &eType, bool bSkipEvents, int nTimeout)
+CNetwork::ResponseType CRTProtocol::Receive(CRTPacket::EPacketType &type, bool skipEvents, int timeout)
 {
     CNetwork::Response response(CNetwork::ResponseType::error, 0);
-    unsigned int nRecvedTotal = 0;
-    unsigned int nFrameSize;
+    unsigned int receivedTotal = 0;
```

Comment:
suggestion: rename `receivedTotal` to `receivedBytesTotal` or something to signify what's being counted.